components/ui/Badge.vue

```diff
@@ -14,20 +14,77 @@
     <template v-else-if="type === 'creator'"><CreatorIcon /> Creator</template>
 
     <!-- Project statuses -->
-    <template v-else-if="type === 'approved'"><ListIcon /> Listed</template>
-    <template v-else-if="type === 'unlisted'"><EyeOffIcon /> Unlisted</template>
-    <template v-else-if="type === 'withheld'"><EyeOffIcon /> Withheld</template>
-    <template v-else-if="type === 'private'"><LockIcon /> Private</template>
+    <template v-else-if="type === 'approved'">
+      <span
+        v-tooltip="'Your project is visible in search and on your profile.'"
+      >
+        <ListIcon /> Listed
+      </span>
+    </template>
+    <template v-else-if="type === 'unlisted'">
+      <span
+        v-tooltip="
+          'Your project is published, and can be accessed with a link, but cannot be seen in search or on your profile.'
+        "
+      >
+        <EyeOffIcon /> Unlisted
+      </span>
+    </template>
+    <template v-else-if="type === 'withheld'">
+      <span
+        v-tooltip="
+          'Your project is accessible by link but is not published in search. See the moderation message for more details.'
+        "
+      >
+        <EyeOffIcon /> Withheld
+      </span>
+    </template>
+    <template v-else-if="type === 'private'">
+      <span v-tooltip="'Your project is only visible to members of your team.'">
+        <LockIcon /> Private
+      </span>
+    </template>
     <template v-else-if="type === 'scheduled'">
-      <CalendarIcon /> Scheduled
+      <span
+        v-tooltip="
+          'Your project will be automatically published at your chosen time and date.'
```

Comment:
```suggestion
          'Your project will be automatically published at the chosen time and date.'
```

"Your" sounds more like it's an initial feature of scheduling, where as "the" sounds more like you have scheduled it.